packages/js-client-rest/src/qdrant-client.ts

```diff
@@ -833,9 +833,7 @@ export class QdrantClient {
     }
 
     /**
-     * Overwrites payload of the specified points
-     * After this operation is applied, only the specified payload will be present in the point.
-     * The existing payload, even if the key is not specified in the payload, will be deleted.
+     * Set only the given payload values on a point.
```

Comment:
Thanks for your finding! Can you please change comment using the exact definition from the documentation:
[Sets payload values for specified points.](https://api.qdrant.tech/api-reference/points/set-payload)

Reply:
Done!